src/containers/Order/styles.module.scss

```diff
@@ -0,0 +1,60 @@
+@import '@scss/mixins';
+
+.adminOrderContainer {
+  @include flex-common(space-evenly, center);
+}
+
+.mb_4 {
+  margin-bottom: 1.5rem !important;
+}
+
+.card {
+  position: relative;
+  box-shadow: 0 0.1rem 0.25rem rgb(0 0 0 / 8%);
+  display: flex;
+  flex-direction: column;
```

Comment:
This can be replaced with `@include flex-col`

Reply:
done